ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.Receive.cs

```diff
@@ -271,6 +284,21 @@ string id
         }
         progress.Report.Merge(converter.Report);
 
+        // track the object type counts as an event before we try to receive
+        // this will tell us the composition of a commit the user is trying to receive, even if it's not successfully received
+        // we are capped at 255 properties for mixpanel events, so we need to check dict entries
+        var typeCountArray = typeCountDict
+          .ToArray()
+          .Select(o => new { TypeName = o.Key, Count = o.Value })
+          .OrderBy(pair => pair.Count)
+          .Reverse()
+          .Take(250);
```

Comment:
Perhaps we can encode this value as a constant inside the Analytics class, just so we avoid having magic numbers

Reply:
SInce the limit in mixpanel is 255, and we might be adding some other default props, I would lower it to 230 or so to be safe, maybe even 200.